src/DotNetCore.CAP.AzureServiceBus/Options/AzureServiceBusOptionsBase.cs

```diff
@@ -123,11 +112,6 @@ public class AzureServiceBusOptions
     /// </remarks>
     public TimeSpan MaxAutoLockRenewalDuration { get; set; } = TimeSpan.FromMinutes(5);
```

Comment:
To maintain parity, I believe we should also add this property to the custom consumer configuration, what do you think?